src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedToArray = sourceString.split(';');
```

Comment:
There is a possibility of having empty strings in the array after splitting by ';'. It would be a good idea to filter out any empty strings before reducing the array.